api/src/controllers/auth.ts

```diff
@@ -232,6 +233,15 @@ router.get(
 			req.session.redirect = req.query.redirect as string;
 		}
 
+		emitAsyncSafe('oauth.provider', {
+			event: 'oauth.provider',
```

Comment:
Lets do the same provider trick here:

```suggestion
		emitAsyncSafe(`oauth.${req.params.provider}`, {
			event: `oauth.${req.params.provider}`,
```